src/main/java/com/conveyal/r5/streets/StreetLayer.java

```diff
@@ -1289,9 +1290,15 @@ public int createAndLinkVertex (double lat, double lon) {
             return -1; // Unlinked
         }
 
-        // TODO give link edges a length.
+        double streetVertexLat = vertexStore.fixedLats.get(streetVertex)/FIXED_FACTOR;
+        double streetVertexLon = vertexStore.fixedLons.get(streetVertex)/FIXED_FACTOR;
```

Comment:
There's a function defined for this operation: com.conveyal.r5.streets.VertexStore#fixedDegreesToFloating(int). We should use the symbolic function name as much as possible to increase readability and decrease the chance of a typo performing the calculation differently in one place. I see that is not the case throughout the code but we should aim for it. 

There is an even more encapsulated way of retrieving these values: `new VertexStore.Vertex(streetVertex).getLat()` and `getLon()`.

Reply:
I will say though that while fixed-point coordinates are a good optimization under certain circumstances, R5 may have outgrown them. Using ints instead of doubles halves the storage space and speeds up certain operations (as well as needing to pull less things in from memory) but our edges now have a lot more data attached to them than they originally did, so the savings may be proportionately smaller. We can address that another time...